code/index.html

```diff
@@ -2,21 +2,186 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div class="logo-container">
+        <a href="#home" class="syzygy-logo">
+          <img
+            src="https://i.ibb.co/KyBfztZ/Untitled-design-5.png"
+            alt="syzygy-logo"
+          />
+        </a>
+      </div>
+      <div class="topnav">
```

Comment:
maybe use 'nav' instead? 
```<div class="topnav">```-> ```<nav class="topnav">```

Reply:
I was deliberating whether I could use nav tag but include the small logo in it still. In the common pitfalls article in W2 it said that <nav> should only include the links, so in the end I made it a div.